packages/ui/src/accounts/components/StakeStep.tsx

```diff
@@ -59,6 +61,7 @@ export const StakeStep = ({
               minBalance={minStake}
               lockType={stakeLock}
               filter={selectAccountFilter}
+              isSmallVariant={isMobile}
```

Comment:
In this case `SelectStakingAccount` component is using small variant because it is located stepper modal which have three columns on `sm` device.
When it is used in normal modal, then it should not be in small variant.
That's why.